lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -193,16 +190,17 @@ class CategoryRenderer {
    */
   renderAuditGroup(group) {
     const groupEl = this.dom.createElement('div', 'lh-audit-group');
-    const summaryEl = this.dom.createChildOf(groupEl, 'div');
-    const summaryInnerEl = this.dom.createChildOf(summaryEl, 'div', 'lh-audit-group__summary');
-    const headerEl = this.dom.createChildOf(summaryInnerEl, 'div', 'lh-audit-group__header');
 
+    const auditGroupHeader = this.dom.createElement('div', 'lh-audit-group__header');
```

Comment:
> In this case I think i'd love it if we just leave a comment like

done

i may try combining these two into the sample template later, in the clean up phase.

Reply:
> i may try combining these two into the sample template later, in the clean up phase.

I was serious about the original TODO :)

>  TODO: group classes shouldn't be reused for clumps.

They don't need to be the same thing, and we don't win anything by creating a grand unified clump-group theorem except a new headache for every point where they're almost but not quite the same thing :P

Seriously, we'll save like two total lines of code and gain some extra conditionals and a bunch of head scratching later about why groups are sometimes what we call groups everywhere else and why they're sometimes something completely different.

Rather than make a comment here, the clean up phase should just `s/group/clump` where that original comment is and be done with it 👍 👍